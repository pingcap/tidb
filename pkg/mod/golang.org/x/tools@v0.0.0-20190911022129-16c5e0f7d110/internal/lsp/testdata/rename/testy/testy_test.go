package testy

import "testing"

func TestSomething(t *testing.T) {
	var x int //@rename("x", "testyX")
	a()       //@rename("a", "b")
}
