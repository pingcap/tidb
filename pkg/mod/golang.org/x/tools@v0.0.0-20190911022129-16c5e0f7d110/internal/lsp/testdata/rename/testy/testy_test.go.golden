-- b-rename --
testy.go:
package testy

type tt int //@rename("tt", "testyType")

func b() {
	foo := 42 //@rename("foo", "bar")
}

testy_test.go:
package testy

import "testing"

func TestSomething(t *testing.T) {
	var x int //@rename("x", "testyX")
	b()       //@rename("a", "b")
}

-- testyX-rename --
testy_test.go:
package testy

import "testing"

func TestSomething(t *testing.T) {
	var testyX int //@rename("x", "testyX")
	a()       //@rename("a", "b")
}

